libgit/rpc.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libgit
+
+import (
+	"os"
+
+	"github.com/keybase/client/go/logger"
+	"github.com/keybase/client/go/protocol/keybase1"
+	"github.com/keybase/kbfs/env"
+	"github.com/keybase/kbfs/kbfsmd"
+	"github.com/keybase/kbfs/libkbfs"
+	"github.com/keybase/kbfs/tlf"
+	"github.com/pkg/errors"
+	"golang.org/x/net/context"
+)
+
+type RPCHandler struct {
+	config libkbfs.Config
```

Comment:
This is the `config` shared with `kbfsfuse` or `kbfsdokan` processes right? Shall we maybe rename it to something like `sharedConfig` to differentiate from the new shortlived `config` generated in the `CreateRepo` handler?

Reply:
Yes and good idea, though I think I'm going to rename the non-shared ones to `gitConfig` to be consistent with having `config` as a struct field everywhere else.

Reply:
sounds good; thanks!